src/desktop_develop.ts

```diff
@@ -294,25 +294,16 @@ export default class DesktopDevelopBuilder {
         logger.info("Build completed!");
 
         if (target.platform === 'darwin') {
-            await fsProm.mkdir(path.join(this.appPubDir, 'install', 'macos'), { recursive: true });
-            await fsProm.mkdir(path.join(this.appPubDir, 'update', 'macos'), { recursive: true });
-
-            for (const f of await getMatchingFilesInDir(path.join(repoDir, 'dist'), /\.dmg$/)) {
-                await copyAndLog(
-                    path.join(repoDir, 'dist', f),
-                    // be consistent with windows and don't bother putting the version number
-                    // in the installer
-                    path.join(this.appPubDir, 'install', 'macos', 'Element Nightly.dmg'),
-                    logger,
-                );
-            }
-            for (const f of await getMatchingFilesInDir(path.join(repoDir, 'dist'), /-mac.zip$/)) {
-                await copyAndLog(
-                    path.join(repoDir, 'dist', f),
-                    path.join(this.appPubDir, 'update', 'macos', f),
-                    logger,
-                );
-            }
+            const distPath = path.join(repoDir, 'dist');
+            const targetInstallPath = path.join(this.appPubDir, 'install', 'macos');
+            const targetUpdatePath = path.join(this.appPubDir, 'update', 'macos');
+
+            await fsProm.mkdir(targetInstallPath, { recursive: true });
+            await fsProm.mkdir(targetUpdatePath, { recursive: true });
+
+            // Be consistent with windows and don't bother putting the version number in the installer
```

Comment:
Arguably, we should add it everywhere?

Reply:
This PR is merely refactoring, also this is develop so growing a million installers is not great